File: bionic-js-tool/testing-code/swift/project-with-host-files/HostProject/host/BjsVehicles/BjsBeautifulVehicles.swift
import Foundation
import Bjs

@objc(BjsBeautifulVehicles)
class BjsBeautifulVehicles : BjsProject {
    
    override class func initialize(_ bjs: Bjs) {
        bjs.loadBundle(BjsBeautifulVehicles.self, "Vehicles")
    }
}

File: bionic-js-tool/testing-code/java/project-with-host-files/HostProject/src/vehicles/java/test/project/host/BjsBeautifulVehicles.java
package test.project.host;

import bionic.js.Bjs;
import bionic.js.BjsProject;
import bionic.js.BjsProjectTypeInfo;

public class BjsBeautifulVehicles extends BjsProject {
    
    @BjsProjectTypeInfo.Initializer
    public static void initialize(Bjs bjs) {
        initProject();
        bjs.loadBundle(BjsBeautifulVehicles.class, "Vehicles");
    }
}